app/controllers/submissions_controller.rb

```diff
@@ -365,7 +368,15 @@ def update_files
                                 ' for assignment ' +
                                 "'#{@assignment.short_identifier}'.")
         end
-
+        #
+        unless students_filename.length < 1
+          if required_assignments != students_filename && @assignment.is_required == true
+            @file_manager_errors[:size_conflict] =
+            "The list of uploaded files does not match the assignment required files"
```

Comment:
Line is too long. [85/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.